scripts/verification/collateral-plugins/verify_oeth.ts

```diff
@@ -0,0 +1,64 @@
+import hre from 'hardhat'
```

Comment:
add this step to `verify.ts`

Reply:
Done in this commit: [b9fae8c](https://github.com/reserve-protocol/protocol/pull/1239/commits/b9fae8c76125ae5756207166ff87f55e66ba5f50)